decidim-meetings/lib/decidim/meetings/feature.rb

```diff
@@ -23,6 +23,8 @@
     meetings.count
   end
 
+  feature.actions = %w(join)
+
   feature.settings(:global) do |settings|
     settings.attribute :announcement, type: :text, translated: true, editor: true
```

Comment:
I think we should provide another setting to actually enable or disable registrations, as not all kinds of events will want that.

Reply:
@josepjaume  @xabier This setting already exists at meeting level so you can enable registrations for a particular event. It is disabled by default.